### DISCUSSION TITLE
About the base image Chainguard image

### DISCUSSION THREAD

**Discussion by xifengdai** [open-policy-agent/community#538] [Community]
i find the question `Fatal glibc error: CPU does not support x86-64-v2` in the pod of k8s in pve vm that cpu type is kvm64, it seems is glibc version. the release note of 0.51.0 is `Update -debug images to use Chainguard images (https://github.com/open-policy-agent/opa/issues/5544) (authored by @charlieegan3)`, But all subsequent types of images are based on it, why is it?

**Comment by charlieegan3** [comment]
Hi @Liwenju, we started using the Chainguard images as it allowed us to build images with fewer surplus vulnerable binaries. We often found OPA images were flagged in scanners for containing openssl, but OPA doesn't use openssl.

I think you're using proxmox? Can you try changing the cpu type to `host` instead?

**Reply by xifengdai** [reply]
ohh, sorry, i haven't seen anything about #5540 yet on release page. All other flavours of images is using Chainguard images now.
yes, i'm using proxmox for k8s node, change the cpu type of vm can be difficult. I'll try something else. thanks

**Reply by charlieegan3** [reply]
Hey, it's in the release notes in https://github.com/open-policy-agent/opa/releases/tag/v0.48.0
